sound/soc/sof/topology.c

```diff
@@ -2304,6 +2308,15 @@ static int sof_dai_load(struct snd_soc_component *scomp, int index,
 	if (!spcm->pcm.playback)
 		goto capture;
 
+	ret = sof_parse_tokens(scomp, &spcm->stream[stream], stream_tokens,
+			       ARRAY_SIZE(stream_tokens), private->array,
+			       le32_to_cpu(private->size));
+	if (ret != 0) {
```

Comment:
doesn't checkpatch propose to just use `if (ret)`? Also below.

Reply:
Agree with @lyakh, please run check patch.

Reply:
It didn't complain, I just manually run it again, no any warnings, anyway, let me change.